scripts/versioning.py

```diff
@@ -0,0 +1,54 @@
+# SPDX-License-Identifier: MIT
+
+import copy
+from typing import Any, Dict, Optional
+
+import packaging.version
+import versioningit
+
+
+def template_fields(
+    *,
+    version: str,
+    description: Optional[versioningit.VCSDescription],
+    base_version: Optional[str],
+    next_version: Optional[str],
+    params: Dict[str, Any],
+) -> Dict[str, Any]:
+    """Implements the ``"basic"`` ``template-fields`` method"""
+    params = copy.deepcopy(params)
+    parsed_version = packaging.version.parse(version)
+    fields: Dict[str, Any] = {}
+    if description is not None:
+        fields.update(description.fields)
+        fields["branch"] = description.branch
+    if base_version is not None:
+        fields["base_version"] = base_version
+    if next_version is not None:
+        fields["next_version"] = next_version

```

Comment:
The template only uses `{version}` and `{version_template}`, are these actually needed here?